kyo-sttp/native/src/main/scala/kyo/PlatformBackend.scala

```diff
@@ -0,0 +1,16 @@
+package kyo
+
+import kyo.Requests.Backend
+import sttp.client3.*
+
+object PlatformBackend:
+    val default =
+        new Backend:
+            val b = CurlBackend()
+            def send[A: Flat](r: Request[A, Any]) =
+                given Frame = Frame.internal
+                def call    = r.send(b)
+                Abort.run[Throwable](IO(call))
```

Comment:
Does this need `Abort.catching`?

Reply:
I don't think so. `Abort.run` already catches